docs/create-packages/Symbol-Packages-V2.md

```diff
@@ -0,0 +1,96 @@
+---
+title: How to publish NuGet symbol packages using the new symbol package format '.snupkg'| Microsoft Docs
+author:
+- cristinamanu
+- kraigb
+ms.author:
+- cristinamanu
+- kraigb
+manager: skofman
+ms.date: 10/30/2018
+ms.topic: reference
+ms.prod: nuget
+ms.technology: null
+description: How to create NuGet packages that contain only symbols to support debugging of other NuGet packages in Visual Studio.
+keywords: NuGet symbol packages, NuGet package debugging, supporting NuGet debugging, package symbols, symbol package conventions
+ms.reviewer:
+- anangaur
+- karann
+- ---
```

Comment:
There is an extra hyphen. All you need is `---`

Reply:
Updated.